packages/react/src/components/FileUploader/next/FileUploader.stories.js

```diff
@@ -20,6 +20,44 @@ import {
 import './FileUploader-story.scss';
 
 const filenameStatuses = ['edit', 'complete', 'uploading'];
+const argTypes = {
```

Comment:
FileUploader is a bit tricky since it has such robust examples but I think we would still want it to follow the pattern of the other components in terms of controls, where only the "Playground" story has controls. To do so, if you don't already know, we would want to remove the passed in `args` from every story besides that one and only define the argtypes for Playground as well.

Reply:
@abbeyhrt  can you explain this a bit more `from every story besides that one and only define the argtypes for Playground as well.`